godtools/App/Share/Data/UserCountersRepository/UserCountersRepository.swift

```diff
@@ -42,6 +49,14 @@ class UserCountersRepository {
     
     func syncUpdatedUserCountersWithRemote() {
         
+        if cancellables.isEmpty == false {
```

Comment:
This is fine as long as ```cancellables``` is strictly used for syncing updated counters.   If other publishers were to sink and store cancellables in the ```cancellables``` ```Set``` then those could potentially be cancelled as well.  For situations like these instead of using ```.store(in: &cancellables)``` I will create an instance variable of ```AnyCancellable?``` that can reference the returned AnyCancellable and cancel the reference.

Reply:
Hey @levieggertcru, I went ahead and moved this entire method into a separate class `RemoteUserCountersSync`.  You're totally right, we need to keep the cancellables from the remote sync completed separated, and just having a separate instance variable still felt dangerous and potentially confusing in the future if we didn't remember which to use where.  So I think it's better encapsulated in a separate class altogether!

Reply:
I like!